SA/design_concept/content/introduction/tool_research/snap.typ

```diff
@@ -2,18 +2,18 @@
 
 Snap! is a block-based programming tool developed by Berkeley University.
 The tool allows users to create imperative programs in a Scratch-like manner.
-A user can create programs to control a cursor in a graphical environment, e.g. setting the cursor to a certain position and drawing a line.
-For that, Snap! offers various pre-defined commands like basic arithmetic operations, cursor-, pen- & sound-controls and some flow controls like "if" blocks.
-For lists, snap offers some control-blocks that allow users to work with lists in a functional manner, as can be seen as the red blocks in @snap_screenshot.
-However, everything else can only be done in an imperative way, making the functional aspects more of an additional feature than a core concept of Snap!.
+A user can create programs to control a cursor in a graphical environment, e.g., setting the cursor to a specific position and drawing a line.
+For that, Snap! offers pre-defined commands like basic arithmetic operations, cursor-, pen- & sound-controls, and flow controls like "if" blocks.
+For lists, snap offers some control-blocks that allow users to work with lists functionally, as seen in the red blocks in @snap_screenshot.
+However, everything else can only be done imperatively, making the functional aspects more of an additional feature than a core concept of Snap!.
 #figure(
     image("../../../static/snap_screenshot.png", width: 90%),
     caption: [Screenshot of a block expression in Snap! @snap-manual]
 )<snap_screenshot>
 
-A user can also create their own block-commands based on already existing commands. This allows users to re-use their code.
+A user can also create their block-commands based on already existing commands. This allows users to reuse their code.
```

Comment:
```suggestion
A user can also create their own block-commands based on already existing commands. This allows users to reuse their code.
```
I think here it's important that the block-commands are the user's block-commands.